### PR TITLE
feat: add tasks for supply chain security

### PR DESCRIPTION
**Pull request by osamamagdy**
Signed-off-by: osamamagdy <osamamagdy174@gmail.com>

### CODE REVIEW

File: tasks/go-plugin/release.yaml [hunk 1]
```diff
@@ -101,6 +101,7 @@ spec:
           script: |
             #!/bin/bash
             source .jx/variables.sh
+            export PATH=$PATH:/go/bin:/usr/local/go/bin:/workspace/source/bin
```
[the remark this answers is not in the document]
Comment (osamamagdy) [reply tasks/go-plugin/release.yaml]: Done

**Review by ankitm123 — APPROVED** [review]

**Pull request merged**